code/game/gamemodes/clock_cult/clock_mobs/_eminence.dm

```diff
@@ -0,0 +1,267 @@
+//The Eminence is a unique mob that functions like the leader of the cult. It's incorporeal but can interact with the world in several ways.
+/mob/camera/eminence
+	name = "\the Emininence"
+	real_name = "\the Eminence"
+	desc = "The leader-elect of the servants of Ratvar."
+	icon = 'icons/effects/clockwork_effects.dmi'
+	icon_state = "eminence"
+	mouse_opacity = MOUSE_OPACITY_OPAQUE
+	move_on_shuttle = TRUE
+	see_in_dark = 8
+	invisibility = INVISIBILITY_OBSERVER
+	layer = FLY_LAYER
+	faction = list("ratvar")
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	var/static/superheated_walls = 0
+
+/mob/camera/eminence/Initialize()
+	if(SSticker.mode.eminence)
```

Comment:
I think you want to invert this check, otherwise you're gonna queue it for qdeletion when it inits (and then skip destruction due to it's Destroy check)

Reply:
`SSticker.mode.eminence` is set in Login, so this is intended to delete any new eminence mobs that somehow get made after one is already assigned.

Reply:
ah okay that makes more sense now that the destroy case is fixed :+1: